luaui/Widgets/cmd_onoff.lua

```diff
@@ -0,0 +1,39 @@
+function widget:GetInfo()
+	return {
+		name = "Onoff for Hound and trajectory",
+		desc = "onoff action will now also switch between Hound weapons and high/low trajectory",
```

Comment:
Should be a generic description, that this applies to all units affected by the cmd_onoffdesc gadget.